src/snapred/backend/recipe/algorithm/LoadGroupingDefinition.py

```diff
@@ -3,10 +3,14 @@
 from mantid.api import AlgorithmFactory, MatrixWorkspaceProperty, PropertyMode, PythonAlgorithm
 from mantid.kernel import Direction
 
+from snapred.backend.log.logger import snapredLogger
 from snapred.backend.recipe.algorithm.MantidSnapper import MantidSnapper
+from snapred.meta.Config import Config
 
 name = "LoadGroupingDefinition"
```

Comment:
We should move away from the use of this variable.  Can you delete this line (L10), and replace it anywhere it is used with `__name__`, as you did in L12?